.github/workflows/release-build-publish.yml

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Build/Publish
+
+on:
+  push:
+    branches: 
+      - release
+    paths:
+      - VERSION
+
+jobs:
+  release-build-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          ref: release
+
+      - name: Login to Quay.io
+        uses: azure/docker-login@v1
+        with:
+          login-server: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+
+      - name: Login to Docker Hub
+        uses: azure/docker-login@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and push patched base images and happy path image
+        run: |
+          export TAG=$(head -n 1 VERSION)
+          /bin/bash arbitrary-users-patch/happy-path/build_happy_path_image.sh --push
+          /bin/bash arbitrary-users-patch/build_images.sh --push
+
+      - name: Build and push images
+        run: |
+          SHORT_SHA1=$(git rev-parse --short HEAD)
+          VERSION=$(head -n 1 VERSION)
+          IMAGE=che-devfile-registry
+          DOCKERFILE_PATH=./build/dockerfiles/Dockerfile
+          docker build -t ${IMAGE} -f ${DOCKERFILE_PATH} --build-arg PATCHED_IMAGES_TAG=${VERSION} --target registry .
```

Comment:
would it make sense to call build.sh here? then we would actually USE it in the release process, rather than just documenting it and telling people to use it for local builds.

Reply:
I'd have to look into it...I just checked what was being called in `cico_release.sh` and used that.

Reply:
Having looked into it, using build.sh is kinda clunky because it doesn't push, and we'd have to call it twice for the short SHA1 tags. Probably not worth it IMO.